website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,102 @@
+## Usage
+
+### When to use
+
+- To show content as needed while keeping the UI organized and visually manageable.
+- To hide long pieces of content with a similar structure.
+
+### When not to use
+
+- To hide basic non-critical information to users. Instead, consider using a [Reveal](/components/reveal).
+- To allow users switch between different content within the same context. Instead, consider using [Tabs](/components/tabs).
+
+### Accordion vs Reveal
+
+While similar in functionality and interaction, the Accordion and Reveal are meant to be used in different scenarios and use cases.
+
+#### When to use the Accordion
+
+- You have multiple sections of content that can be expanded or collapsed.
+
+- Each section of content has its own toggle content, giving users the ability to access specific sections.
+
+- Users need to navigate through different sections of content and selectively expand or collapse sections
+
+- There is a sequential or hierarchical relationship between the different sections.
+
+#### When to use the Reveal
+
+- You have a single piece of content that can be expanded to collapsed to reveal additional details or information.
+
+- The content within the toggle is plain text.
+
+- The content within the reveal is relatively short and simple.
+
+- There is no need for multiple independent sections or hierarchical relationship between different sections.
+
+### Usage examples
+
+!!! Do
+
+Use the Accordion when you need to display content in different sections that have a sequential or hierarchical relationship between them.
+
+![Accordion with basic content](/assets/components/accordion/usage-do.png)
+!!!
+
+!!! Dont
+
+Don’t use the Accordion when you need to display a single piece of content that can expand and collapse. Use the [Reveal](/components/reveal) instead.
+
+![Accordion with basic content](/assets/components/accordion/usage-do-not.png)
+!!!
+
+!!! Do
+
+Use the Accordion when the toggle or the content is relatively complex.
+
+![Accordion with complex content](/assets/components/accordion/usage-do-complex.png)
+!!!
+
+## Toggle content
+
+The toggle accepts many different types of content, from text-based content to nested HDS components or custom content.
+
+![Example of toggle content](/assets/components/accordion/toggle-content-text.png =690x*)
+
+### Generic content
+
+![Example of toggle content](/assets/components/accordion/toggle-content-custom.png =690x*)
+
+### Interactive content
+
+Use `isInteractive` when nesting interactive content on the toggle.
```

Comment:
+1 to @didoo's comment above. We should help consumers understand when to use this variant over the default one. 

```suggestion
Use `containsInteractive` when nesting interactive content in the toggle.
```